data/publiccloud/log_instance.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

Reply:
Actually I don't need to be compatible, why making my live harder :)?

Reply:
ok